docs/user/tipsandtricks.md

```diff
@@ -143,14 +143,14 @@ In conda-forge, some packages are available with GPU support. These packages not
 In order to override the default behavior, a user can set the environment variable `CONDA_OVERRIDE_CUDA` like below to install TensorFlow with GPU support even on a machine with CPU only.
 
 ```shell-session
-CONDA_OVERRIDE_CUDA="11.2" conda install "tensorflow==2.7.0=cuda112*" -c conda-forge
+CONDA_OVERRIDE_CUDA="<CUDA version>" conda install "tensorflow==2.7.0=cuda<version>*" -c conda-forge
 # OR
-CONDA_OVERRIDE_CUDA="11.2" mamba install "tensorflow==2.7.0=cuda112*" -c conda-forge
+CONDA_OVERRIDE_CUDA="<CUDA version>" mamba install "tensorflow==2.7.0=cuda<version>*" -c conda-forge
 ```
 
 :::note
 
-You should select the cudatoolkit version most appropriate for your GPU; currently, we have "10.2", "11.0", "11.1", and "11.2" builds available, where the "11.2" builds are compatible with all cudatoolkits>=11.2. At the time of writing (Mar 2022), there seems to be a bug in how the CUDA builds are resolved by `mamba`, defaulting to `cudatoolkit==10.2`; thus, it is prudent to be as explicit as possible like above or by adding `cudatoolkit>=11.2` or similar to the line above.
+See the [relevant CUDA user guides](https://github.com/conda-forge/cuda-feedstock/blob/main/recipe/README.md) for more information.
```

Comment:
Here I remove all mention of `cudatoolkit` (outdated) by just pointing to the user guides. There are still mentions of `cudatoolkit`

```console
$ git grep "cudatoolkit" -- docs/
docs/maintainer/knowledge_base.md:2081:- [cudatoolkit-feedstock](https://github.com/conda-forge/cudatoolkit-feedstock)
docs/maintainer/knowledge_base.md:2089:1. Add the `cudatoolkit` packages in `cudatoolkit-feedstock`.
```

but I'd prefer those to get handled in a different PR, given the nature of the section they appear in.

Reply:
Meaning this section?

https://github.com/conda-forge/conda-forge.github.io/blob/3cc7737a0f5aca9a07608d98ccafff92826fc824/docs/maintainer/knowledge_base.md?plain=1#L2123

That is also all outdated. So think it would be ok to drop

Current process is NVIDIA updates all of the CUDA Toolkit feedstocks and afterwards we can [bump `conda-forge-pinning`'s `cuda_compiler_version`]( https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/d096296da5a195ff56fca18758c528c5aa7ee6da/recipe/conda_build_config.yaml#L62 )

The other feedstocks it references are either CUDA 11 only (`nvcc` & `cudatoolkit`) or no longer affected by CUDA updates (`docker-images` & `conda-forge-ci-setup`)